.travis.yml

```diff
@@ -8,23 +8,9 @@ install:
   - pip install git+https://github.com/eregs/regulations-parser.git@4.0.0#egg=regparser
   - npm install -g grunt-cli bower
   - gem install sass
-env:
-  global:
-    # Set cloud.gov credentials
-    - secure: "nCrR9IDVhRWOdRwxFsS847e8sBzcNxg/Pfw5/Zcl423NjEGI5QDQrcA5MbIBjdqrYKbHBMiWM6dS47YWxg9ZHi0joqhOmVJXg0cuvX8pa+loS66fUBDgeqWjTNW9mqiNuHkxRrV7YC6XLMw49gc+SI1wcK2gqC3zcsCOnGA1oviHHvs6ZOPVNkhnJFdFk6VmkUWdHUP2GfV7VVSXJ2T6Dq3NPbfRH34V1wypnf4seVTOjMkZnIa7OO5Uc8CYjNCLAnyT3uCg0k65xIDIxwItN9o08lGuphKZrfHA6zxbFK4McBJO6dO8H4yKG/fNWpr8k7vISE9OMHSJVr8e9RoVbWeDGQtSQkc9XuYQyX2yDYeQjH5qTTrn6FZKj4N+KSZEmcUQ8/HdHHEsFaacdGUGh5aJK76+rjjYsDK+P5saXwdM5eWRuGzHp5Grgn3GMQnz5nBoaXTdyuQFNuBHcfpkePtrYTUP/+7vzyaHsamAtq0RYmBPWARiZyujemaiDUU0vMn2Hrty1FsVJm/NneTeLrpYeTw3J/KWWo3h9VnqHB1OB1ekK9aBadAlXgWQj+Cn/uNzRpZGacWTd+YqbYPjPaV1ASezMMSf1zHzju/tyZWBGPwfMpp3l6Zx82CSO3A1F3UmoKKGyaAGcvGyyAA7IGsPb05Sxl0O+9SA0RgV6IU="
-    - secure: "WV/5LaDd6OED6MkIOdSxbhOoYTCkVPO19Qz14IzkIBfp/ej6YFMW0Ksc40cBSR9I5xjaB7lLNgYhBN/CkunS/O6AAYlPrj/AH9C5+7dm8T+kbsgILtey4Fe/PUi/MOkyikNiLWHfZiagCr1isF7K3oNpgAhjXwanVdMAl48hgQ7xjDhb39VIDtPLU86YUWuLava3r1xtRIR7ruhBsU1WXSAktd/LcSgVk9UZFcFacXkZk6WyYjDs3vsCXdpm/6YiFMl/2dzm8z6paXwh2hsjO46su0AvOyaK5EU3PjONZEERbYAbn0Tu9S4B9kcG6EvDT6hVaJ5F5KITZP+F0nLR/YQ91S+gsN43bqc6sklc1mtMe7PLlziNeQ6yWZ4HjSztd6wAwG7k5n6WU8+gHi7O845AWqQzff4q14orcev5l8JekSZaK1oYqv9jIexnk1QgU4l2G/NcuUH9MY+WUJFCF0GKTjoEUPxL98eJjdP+FKYj93fA72WwSUg1lfrGtEhtGMpyuQeiJxksPxeVUCdfxxXqUVFPOEqHU6XDBhjbcDSiM6qbEjLXzkw3cxpppD18UN7fkGL4WqZUy5ID9hY83pX1t6/Y8PEOdOyQyjq0ItXvO1RJPDLhzBFjhccr5lAGO/+tvYEEvisL3aivYw+imMGKCXSi9fDxFy5MeanCGGs="
 script:
   - python manage.py migrate --fake-initial
   - python manage.py compile_frontend
   - py.test
   - flake8 atf_eregs eregs_extensions
   - bandit -r .
-before_deploy:
-  - export PATH=$HOME:$PATH
-  - travis_retry curl -L -o $HOME/cf.tgz "https://cli.run.pivotal.io/stable?release=linux64-binary&version=6.15.0"
-  - tar xzvf $HOME/cf.tgz -C $HOME
-  - travis_retry cf install-plugin autopilot -f -r CF-Community
-deploy:
-  provider: script
-  script: ./deploy-travis.sh
-  skip_cleanup: true
```

Comment:
Yes, we'll need to get a new deploy user set up, export the env vars, etc. I'm glad to get to a stable state now, though.